app/settings.py

```diff
@@ -75,14 +75,20 @@
 # Database
 # https://docs.djangoproject.com/en/4.2/ref/settings/#databases
 
+# DATABASES = {
+#     "default": {
+#         "ENGINE": environ.get("DB_ENGINE"),
+#         "NAME": environ.get("DB_NAME"),
+#         "USER": environ.get("DB_USER"),
+#         "PASSWORD": environ.get("DB_PASSWORD"),
+#         "HOST": environ.get("DB_HOST"),
+#         "PORT": environ.get("DB_PORT"),
+#     }
+# }
 DATABASES = {
     "default": {
-        "ENGINE": environ.get("DB_ENGINE"),
-        "NAME": environ.get("DB_NAME"),
-        "USER": environ.get("DB_USER"),
-        "PASSWORD": environ.get("DB_PASSWORD"),
-        "HOST": environ.get("DB_HOST"),
-        "PORT": environ.get("DB_PORT"),
+        "ENGINE": "django.db.backends.sqlite3",
+        "NAME": BASE_DIR / "db.sqlite3",
```

Comment:
Won't this cause problems when we come to the point where we want to build/deploy a charm for this?

Reply:
That change shouldn't be there. My bad